conf/messages.en

```diff
@@ -2454,7 +2468,11 @@ employerContributions.MemberList.add.hidden.text = employer contributions for {0
 employerContributions.MemberList.change.hidden.text = employer contributions for {0}
 employerContributions.MemberList.remove.hidden.text = employer contributions for {0}
 employerContributions.MemberList.viewOnly.title = Employer contributions
-employerContributions.MemberList.viewOnly.heading = Employer contributions
+
+employerContributions.MemberList.viewOnly.noContributions = Employer contributions
+employerContributions.MemberList.viewOnly.singular = 1 Employer contribution
+employerContributions.MemberList.viewOnly.plural = {0} Employer contributions
```

Comment:
Sorry @Arei1997 , it's nitpicking,  but should it be `{0} employer contributions` and `1 employer contribution`?

Reply:
Correct, Sorry about that - Added the changes now